service/elastigroup/providers/aws/aws.go

```diff
@@ -1188,6 +1189,25 @@ type DeallocateStatefulInstanceInput struct {
 
 type DeallocateStatefulInstanceOutput struct{}
 
+type Logging struct {
+	Export *Export `json:"export,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+type Export struct {
+	S3 *S3 `json:"s3,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+type S3 struct {
+	ID *string `json:"id,omitempty"`
```

Comment:
modify it to Id (camelcase) instead of ID

Reply:
Updated to Id.